### ISSUE TITLE
Mastering Plone 6: Notes from a newbie

### ISSUE THREAD

**Issue by m0lentum** [plone/training#774]
I participated in Plone Midsummer Sprint 2023 with no previous Plone experience and spent the week working through the Mastering Plone 6 Development course. I wrote down some notes on things that caused me trouble and was asked to post them as an issue here.

The biggest problems I had were because I installed Plone with the instructions at https://6.docs.plone.org/install/install-from-packages.html instead of those in the course, which made a differently structured project (no buildout on the backend side). There was also some confusion about what should be done in Volto vs. classic UI and in an add-on vs. directly in the backend. Overall it went well and I wasn't stuck on anything for very long. I made it to chapter 25 during the sprint.

Here are the notes I wrote:
- section 9.3.1.2: It took me a little while to realize `venv/bin/zconsole debug instance/etc/zope.conf` was a console command, since it was formatted like the file path in the previous exercise
- looks like section 9.10 is obsolete and Volto can now add content rules
- section 9.9 and onwards has a lot of seemingly unfinished bullet point lists that are hard to follow
- section 11.2 talks about how to use portlets, but I don't really understand what a portlet _is_ (maybe not relevant because they're not in Volto?)
- section 12.1: I could not find the directory `omelette` in the example code (edit: found it later, it wasn't there because I hadn't run `yarn`)
- section 16.3.1: I couldn't find a buildout.cfg file in my environment installed with the instructions at https://6.docs.plone.org/install/install-from-packages.html. I used the instructions at https://6.docs.plone.org/install/manage-add-ons-packages.html to install an add-on instead. This also caused much confusion when making a custom add-on in chapters 17 and 19
- section 16.4: I first made the form in Volto and couldn't find the "actions" menu. Took me a while to realize it didn't exist in Volto and I was supposed to be in the classic UI
- section 16.5: It's unclear to me how Mosaic relates to Volto, seems it's classic UI only?
- section 18.5: The instructions link to classic UI but screenshots are from Volto
- section 19.1: The path `backend/src/ploneconf.site/sources/ploneconf/site/profiles/default/types.xml` is wrong, instead of `sources` it should be `src` (this one took a while to figure out 😅)
- chapter 22: The variable `{content.type_of_talk.title}` in the example code doesn't exist, `content.type_of_talk` is a string
- chapter 22: The `Helmet` element disappears from the example code after it's first added
- section 23.3: It's unclear where this behavior goes (backend or the custom add-on)

**Comment by ksuess** [comment]
Thank you for your feedback!

**Comment by Vivek-04022001** [comment]
@m0lentum , Thank you for your notes.

**Comment by stevepiercy** [comment]
@ksuess can this issue be closed?

**Comment by ksuess** [comment]
Thank you for your feedback. Even if it's a late response, your feedback found it's way to the training documentation. I made an overhaul last summer/autumn with lots of corrections, enhancements, and new chapters.
Thank you for your contribution @m0lentum !
